incubator/hnc/config/rbac/role.yaml

```diff
@@ -13,13 +13,10 @@ rules:
   verbs:
   - create
   - delete
-  - deletecollection
   - get
-  - impersonate
   - list
   - patch
   - update
-  - use
```

Comment:
Do we need to exclude role.yaml to a single PR by itself? @adrianludwin

Reply:
That would be nice - Yiqi, do you mind submitting that PR? Sophie, would you please exclude this file from your commit?

Reply:
sure will do shortly.

Reply:
Reverted this file. Thanks Yiqi for the clean up!